test/template-instance.js

```diff
@@ -216,4 +216,84 @@ describe('template-instance', () => {
       })
     })
   })
+
+  describe('custom processors', () => {
+    describe('createCallback', () => {
+      it('is called on construction, if present', () => {
+        const template = document.createElement('template')
+        template.innerHTML = `<div>{{a}}</div>`
+        let createCallCount = 0
+        new TemplateInstance(
+          template,
+          {a: true},
+          {
+            createCallback() {
+              createCallCount += 1
+            },
+            processCallback() {
+              return
+            }
```

Comment:
Should the `processCallback` not have a default if it isn't provided? Does it maybe have a default and we are explicitly overriding it?

Reply:
It does not have a default. It is expected to exist. Other than for tests it's quite pointless to create a template instance which doesn't process anything!

The whole processor argument does have a default, though.